.github/workflows/prod.yml

```
@@ -0,0 +1,112 @@
+              if curl -sf "http://localhost:${STANDBY_PORT}/health-check"; then
+                HEALTHY=true
+                echo "** Health-Check 성공: ${STANDBY_PORT} 포트 응답 OK"
+                break
+              else
+                echo "** Health-Check 실패, 재시도 중... (${i}/5)"
+                sleep 5
+              fi
+            done
+            if [ "$HEALTHY" = true ]; then
+              echo "** Health-Check 성공: ${AFTER_NAME} 컨테이너 정상 작동"
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)